### PR TITLE
Onboard Camera System.md

### PR DESCRIPTION
**Pull request by KrauseOlly**
## Describe your project in 1-3 sentences

I plan on building a camera system that will be able to record and broadcast (using analog fpv equipment) back to spectators so they can watch the sport live from various views

## Hack Club Slack handle

KrauseOllyKrause

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

File: KrauseOlly.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+name: "@yourGitHubUsername"
```
Comment (bellesea) [inline KrauseOlly.md:2]:
```suggestion
name: "@KrauseOlly"
```

**Review by bellesea — CHANGES_REQUESTED** [review]
hiii @KrauseOlly this is super cool! love that you're desiging your own camera housing! i'm wondering, how will your camera be sending information to your goggles?

also, you mentioned having various views but i see that you're purchasing only one camera? how many cameras do you plan on using?

thank you sm!

**Comment by KrauseOlly** [comment]
> hiii @KrauseOlly this is super cool! love that you're desiging your own camera housing! i'm wondering, how will your camera be sending information to your goggles?
> 
> also, you mentioned having various views but i see that you're purchasing only one camera? how many cameras do you plan on using?
> 
> thank you sm!

Hello @bellesea to your first question the camera I am purchasing has a built in transmitter and I already have a receiver for the goggles at home. As for the various views for now I just plan on getting one camera working on the front panel of my go kart however later down the line when I have a bit more money I plan on adding a reverse and side cameras to the go kart. Hope this answers your question.

**Comment by bellesea** [comment]
> > hiii @KrauseOlly this is super cool! love that you're desiging your own camera housing! i'm wondering, how will your camera be sending information to your goggles?
> > also, you mentioned having various views but i see that you're purchasing only one camera? how many cameras do you plan on using?
> > thank you sm!
> 
> Hello @bellesea to your first question the camera I am purchasing has a built in transmitter and I already have a receiver for the goggles at home. As for the various views for now I just plan on getting one camera working on the front panel of my go kart however later down the line when I have a bit more money I plan on adding a reverse and side cameras to the go kart. Hope this answers your question.

ahhh coool, thank you so much for clarifying!

**Review by bellesea — APPROVED** [review]
@KrauseOlly thanks so much for sharing this project idea!! it's sooo cool and good luck w/ the designing of the camera housing, hope you're able to build one that gets you the best footage :)

an awesome member of @hackclub/bank-ops will be reaching out w/ next steps soon (likely tmrw EST). thanks!

**Review by MelSmith104 — COMMENTED** [review]
hey @KrauseOlly omg this project looks amazing!!! im misread at first and thought you were building a go kart but this entire project sounds so cool and unique! i can't wait to see how you build it! dm'ing you on slack right now to talk about next steps :)

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**